pkg/pillar/cmd/zedagent/handlecontent.go

```diff
@@ -65,6 +65,7 @@ func parseContentInfoConfig(ctx *getconfigContext,
 		contentConfig.ContentSha256 = cfgContentTree.GetSha256()
 		contentConfig.MaxDownloadSize = cfgContentTree.GetMaxSizeBytes()
 		contentConfig.DisplayName = cfgContentTree.GetDisplayName()
+		contentConfig.ObjType = types.AppImgObj
```

Comment:
Why do we need to set this? Trying to get rid of ObjType as much as possible so need to understand why it needs to be added in new places.

Reply:
Actually we should keep it here since it is identifying the client i.e., where to publish the Status.